pyproject.toml

```diff
@@ -16,7 +16,7 @@ linkml-runtime = ">=1.5.3"
 linkml-renderer = ">=0.3.0"
 networkx = ">=2.7.1"
 sssom-schema = ">=0.11.0"
-sssom = ">=0.3.38"
+sssom = ">=0.4.0"
```

Comment:
`sssom==0.4.0rc1` has been released. Updating this dependency will get you further in QC tests.

Reply:
@hrshdhgd thanks, it looks like kgcl-schema has some conflicting dependencies now. How do we unpin a maximum version of `curies` for that?

Reply:
kgcl-schema = ">=0.6.0" should fix it.

Reply:
Thank you!

Reply:
As per Chris:
> potentially <0.5.0 if more breaking changes are expected

May need to be "^0.4.0"